schema_queries_test.go

```diff
@@ -0,0 +1,35 @@
+//go:build integration && scylla
+// +build integration,scylla
+
+package gocql
+
+import (
+	"testing"
+	"time"
+)
+
+func TestSchemaQueries(t *testing.T) {
+	cluster := createCluster()
+
+	fallback := RoundRobinHostPolicy()
+	cluster.PoolConfig.HostSelectionPolicy = TokenAwareHostPolicy(fallback)
+
+	session := createSessionFromCluster(cluster, t)
+	defer session.Close()
+
+	assertTrue(t, "keyspace present in schemaDescriber", session.schemaDescriber.cache["gocql_test"].Name == "gocql_test")
+}
+
+func TestLowTimeout(t *testing.T) {
+	cluster := createCluster()
+	cluster.MetadataSchemaRequestTimeout = 1 * time.Nanosecond
```

Comment:
It is 1 ns, and I would expect it to always fail with such a low timeout

Reply:
One day Scylla will be fast enough to cause this test to fail :) but till then it's very safe.

Reply:
This will fail, but due to a different problem. Unfortunately with using timeout you can go only down to 1 millisecond. It will look like this:
```
cqlsh> SELECT * FROM system.versions USING TIMEOUT 1ns;
InvalidRequest: Error from server: code=2200 [Invalid query] message="Timeout values cannot have granularity finer than milliseconds"
```

Reply:
you are right I will remove this test and change the unit to ms in timeoutClause.